### PR TITLE
add libp2p day

### PR DESCRIPTION
**Pull request by p-shahi**

### CODE REVIEW

**Comment by DeniceZ** [comment]
@rmochido1 libp2p wants to confirm venue as Time Out Market for their event. Can you work with @p-shahi to finalize the venue reservation?

File: events/libp2p-day-event.toml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+days = 1
+
+# the event times (shows up in the event card)
+times = "To be confirmed"
+
+# the event venue name (will show up on the event card - TODO)
+venueName = "Time Out Market"
+
+# the event venue address (will show up on a map -- TODO)
+venueAddress = "Av. 24 de Julho 49, 1200-479 Lisboa, Portugal"
```
Comment (p-shahi) [inline events/libp2p-day-event.toml:28]:
```suggestion
venueName = ""

# the event venue address (will show up on a map -- TODO)
venueAddress = ""
```

File: events/libp2p-day-event.toml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+days = 1
+
+# the event times (shows up in the event card)
+times = "To be confirmed"
+
+# the event venue name (will show up on the event card - TODO)
+venueName = ""
+
+# the event venue address (will show up on a map -- TODO)
+venueAddress = ""
```
Comment (p-shahi) [inline events/libp2p-day-event.toml:28]:
```suggestion
venueName = "Convento do Beato"

# the event venue address (will show up on a map -- TODO)
venueAddress = "Alameda do Beato 40, 1950-042 Lisboa, Portugal"
```

**Review by laeccles — APPROVED** [review]
Your pull request has been merged!

**Pull request merged**